Component Patterns Updates (#1463)
ember-learn/guides-source: guides/v3.17.0/in-depth-topics/patterns-for-components.md

This guide assumes that you have read the [component guides](../../components/)
and are familiar with how Ember components work. We'll cover components in more

> depth in this section, including additional functionality and patterns.

Reviewer: The existing phrase "additional functionality" feels vague to me. I think it'd be also good to explicitly mention in the intro that component patterns will be presented at the end.

What do you think of:

```suggestion
depth in this section. At the end, we'll present recommended component patterns.
```

Picking different components to render in response to the data allows you to
have different template and behavior for each case. The `{{component}}` helper
is a powerful tool for improving code modularity.

> ---

Reviewer: Rather than using a line divider, can we make an `h2`-level section so that people can link to it?

```suggestion
## Learn More
```

> To avoid this guide from going on too long we've built a site entirely around
> [ember-component-patterns that you can check out here](https://emberjs-1.gitbook.io/ember-component-patterns/).

Reviewer: For brevity, what do you think of:

```suggestion
To keep this guide concise, we built a separate site for [component patterns in Ember](https://emberjs-1.gitbook.io/ember-component-patterns/).
```

> This project covers individual component patterns in greater detail complete 
> with anti-patterns and accessibility in mind.

Reviewer: ```suggestion
This project also addresses anti-patterns and accessibility for components.
```
